domain/network/network.py

```diff
@@ -0,0 +1,200 @@
+from dataclasses import asdict, dataclass
+from datetime import datetime, timedelta
+from typing import List, Set
+
+from dacite import from_dict
+
+from domain.network.node import Node, NodeState, Peer
+
+
+def uptime_to_up_since(uptime: float) -> int:
+    """ Calculate time when server went up considering server uptime. Remove precision miliseconds to avoid differences
+        between two instances with the same data calculated in different milliseconds
+
+    :param uptime: Time for how long the server is up in seconds
+    :type uptime: float
+    :return: Calculated datetime when server went up in second (rounded down)
+    :rtype: int
+    """
+    up_since_date = datetime.now() - timedelta(seconds=uptime)
+    return int(datetime.timestamp(up_since_date))
+
+
+@dataclass
+class AggregatedPeer:
+    """Aggregated data of a Peer retrieved by requesting a Node
+
+    :param id: id of peer
+    :type id: str
```

Comment:
I don't think we need the `type` anymore since we are using type annotations in the code. It will probably get outdated in the future.

Reply:
Should we remove it? I don't have an opinion about it yet. As we're not generating any doc and the IDE is pretty much fine with the type hints, i think it doesn't matter to me, we can remove or keep it.